src/components/views/ViewMilestone.jsx

```diff
@@ -647,18 +647,22 @@ const ViewMilestone = props => {
                     )}
 
                     <div id="donations" className="spacer-top-50">
-                      <div className="section-header">
-                        <h5>{donationsTitle}</h5>
-                        {isActiveMilestone() && <DonateButton {...donateButtonProps} />}
-                      </div>
-                      <DonationList
-                        donations={donations}
-                        isLoading={isLoadingDonations}
-                        total={donations.length}
-                        loadMore={loadMoreDonations}
-                        newDonations={newDonations}
-                        useAmountRemaining
-                      />
+                      {milestone.status !== 'Proposed' && (
```

Comment:
Use variable in comparing rather than values like here:
https://github.com/Giveth/giveth-dapp/blob/785acfc69ac5e0a0964accfecdf1f9d2f29b7cdd/src/services/CampaignService.js#L134